spool_2/regex_spec.rb

```diff
@@ -4,73 +4,74 @@
 
   let(:phrase) { "It was a bright cold day in April, and the clocks were striking thirteen." }
 
-  it "matches using =~" do
+  it "matches using match" do
```

Comment:
Okay. I don't think the differences between `=~` and `match` affect any of these specs so it should be fine.